cilium/cilium.io
# add eficode to the top section of the adopters list

**This PR:**
Adds eficode to the Adopters page;

**Steps to test**

- Open the [Adopters page](https://deploy-preview-286--cilium.netlify.app/)
- Confirm that everything looks and works as expected
